web/libs/editor/src/stores/Comment/Comment.js

```diff
@@ -115,6 +113,32 @@ export const Comment = CommentBase.named("Comment")
       self.setEditMode(false);
     });
 
+    const update = flow(function* (props) {
+      if (self.isPersisted && !self.isDeleted && !self.isUpdating) {
+        self.isUpdating = true;
+        const [result] = yield self.sdk.invoke("comments:update", {
+          id: self.id,
+          ...props,
+        });
+        if (result.error) return;
+        const data = camelizeKeys(result);
+        applySnapshot(self, data);
+        self.isUpdating = false;
+      }
+    });
+
+    function setRegionLink(region) {
+      const regionRef = {
+        regionId: region.cleanId,
+      };
+      self.update(snakeizeKeys({ regionRef }));
```

Comment:
things like this should be on a same level to incapsulate knowledge. move this transformation to the `update()` which also uses reversed transform.